src/components/SwipeButtons.tsx

```diff
@@ -2,31 +2,29 @@ import React from 'react'
 import { View, Text, StyleSheet, TouchableOpacity, Button } from 'react-native'
 import { MaterialCommunityIcons } from '@expo/vector-icons';
 import { Feather } from '@expo/vector-icons';
+import { SwipeButtonsParamList } from "../../types"
 
-export type SwipeButtonsParamList = {
-    onPress: void
-}
 
-export default function SwipeButtons({onPress}: {onPress: SwipeButtonsParamList}) {
+ const SwipeButtons = ({handleOnPressLeft, handleOnPressRight}: SwipeButtonsParamList) => {
     return (
         <View style={styles.container}>
-            <TouchableOpacity style={styles.button} onPress={() => {return onPress}}>
+            <TouchableOpacity style={styles.button} onPress={() => handleOnPressLeft()}>   
                 <Feather name="x-circle" size={24} color="black" />
             </TouchableOpacity>
-            <TouchableOpacity style={styles.button} onPress={() => {return onPress}}>
-                <MaterialCommunityIcons name="silverware-fork-knife" size={24} color="black"/>
+            <TouchableOpacity style={styles.button} onPress={() => handleOnPressRight()}>
```

Comment:
onPress can simply be ```onPress={handleOnPress}``` instead of ```onPress={() => handleOnPress()}```. Unless you plan to pass the event argument. Just a thought to clean it up a bit.